service-registry/src/main/java/io/servicecomb/serviceregistry/api/registry/Microservice.java

```diff
@@ -57,17 +57,17 @@
   private Map<String, String> properties = new HashMap<>();
 
   @JsonIgnore
-  private MicroserviceInstance intance;
+  private MicroserviceInstance instance;
 
   public Microservice() {
   }
 
-  public MicroserviceInstance getIntance() {
```

Comment:
Please add the deprecated annotation and call the new fixed method to avoid the API broken issue.